kinflate: added version command

droot opened the pull request:
add version package. Will be using it in container builder to embed version in all the releases. It will help us during debugging any external issues.

mengqiy commented on cmd/kinflate/version/version.go:
```diff
@@ -0,0 +1,67 @@
+	}
+}
+
+func (v Version) Print() {
```
I'd like to see it prints to an `io.writer`.
IOW, `NewCmdVersion` takes another argument `stdOut io.writer`.

droot replied:
Done

droot commented:
@mengqiy Thanks for the quick review. Addressed the comment about the writer.
Example output

```
Version: {KinflateVersion:1.6 GitCommit:$Format:%H$ BuildDate:1970-01-01T00:00:00Z GoOs:unknown GoArch:unknown}
```

In our releases, all the fields will be filled in properly.

monopole commented:
/approve

monopole commented:
/lgtm
